### PR TITLE
#83 jquery convert to pure javascript changes 

### PR DESCRIPTION
**Pull request by AgriyaDev5**
## PR description

 jquery convert to pure javascript changes

### CODE REVIEW

File: src/editor/extensions/ext-arrows/ext-arrows.js [hunk 1]
```diff
@@ -6,6 +6,9 @@
  * @copyright 2010 Alexis Deveria
  *
  */
+ import {
+  $id, $qa, $qq
+} from '../../../svgcanvas/utilities.js';
```
Comment (jfhenon) [inline src/editor/extensions/ext-arrows/ext-arrows.js]: I would like to make svgedit modules not dependant on internal source code of svgcanvas. Ultimately, I think we could have svgcanvas as a separate npm modules. 
However, these utilities functions have been attached to the smain vgcanvas object so you could reuse them only with the import of svgcanvas.js

``````
import SvgCanvas from '../svgcanvas/svgcanvas.js';
const {$id, $qa, isNullish, encode64, decode64, blankPageObjectURL} = SvgCanvas;
`````

In fact, if you have access to the svgcanvas runtime object (or the editor object), I didn't test but it should be possible to access to the function without import.

File: src/editor/Editor.js [hunk 2]
```diff
@@ -959,7 +961,9 @@ class Editor extends EditorStartup {
   * @returns {void} Resolves to `undefined`
   */
   cancelOverlays (e) {
-    $('#dialog_box').hide();
+    if($id("dialog_box") != null){
```
Comment (jfhenon) [inline src/editor/Editor.js]: maybe you can use the new ? operator to make this more readable. 
$id("dialog_box")?.style.display = 'none'

File: src/editor/extensions/ext-arrows/ext-arrows.js (hunk 1 above)
Comment (AgriyaDev5) [reply src/editor/extensions/ext-arrows/ext-arrows.js]: Modified now please check it.

File: src/editor/Editor.js (hunk 2 above)
Comment (AgriyaDev5) [reply src/editor/Editor.js]: Modified now please check it.

**Review by jfhenon — APPROVED** [review]

**Pull request merged**